projects/igniteui-angular/src/lib/input-group/input-group.component.ts

```diff
@@ -174,12 +207,23 @@ export class IgxInputGroupComponent extends DisplayDensityBase implements IgxInp
      * @hidden
      */
     @HostListener('click', ['$event'])
-    public onClick(event) {
-        if (!this._supressInputAutofocus) {
+    public onClick(event: MouseEvent) {
+        // if the click target is not the input, e.g. click on prefix, focus the input.
+        if (event.target !== this.input.nativeElement || !this._supressInputAutofocus) {
             this.input.focus();
         }
     }
 
+    /** @hidden */
+    @HostListener('pointerdown', ['$event'])
+    public onPointerDown(event: PointerEvent) {
+        // if the pointer is not on the input, e.g. pointer is on the prefix, call preventDefault.
+        // This will prevent blur of the input and focusing of the element under the pointer.
+        if (event.target !== this.input.nativeElement || this._supressInputAutofocus) {
```

Comment:
So if the target is not the input, this will always prevent (which is the goal of the change)? Don't see how `supressInputAutofocus` fits in here just for clicks on the input itself

Reply:
fixed